src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/loadbalancer/ApacheEc2LoadBalancerFormula.kt

```diff
@@ -56,6 +60,16 @@ internal class ApacheEc2LoadBalancerFormula : LoadBalancerFormula {
         )
     }
 
+    override fun updateJiraConfiguration(ssh: Ssh, unpackedProduct: String) {
+        ssh.newConnection().use { shell ->
+            Sed().replace(
+                shell,
+                "bindOnInit=\"false\"",
+                "bindOnInit=\"false\" scheme=\"http\" proxyName=\"${uri?.host}\" proxyPort=\"80\"",
```

Comment:
IMO it's better to fail on null here instead of allowing to silently break the configuration.

Reply:
Currently it would produce `proxyName="null"`